cmd/test/main.go

```diff
@@ -158,7 +159,12 @@ func New(ctx *TestCommandContext) *cobra.Command {
 				return err
 			}
 
-			testCommandFlags := TestCommandFlags{Output: outputFlag, K8sVersion: k8sVersion, IgnoreMissingSchemas: ignoreMissingSchemas, PolicyName: policy, SchemaLocations: schemaLocations, OnlyK8sFiles: onlyK8sFiles}
+			notStrict, err := cmd.Flags().GetBool("not-strict")
```

Comment:
as mentioned in our issue discussion, what do you think about changing the flag to `--schema-not-strict`